metadata-ingestion/src/datahub/ingestion/api/source_helpers.py

```diff
@@ -119,14 +118,9 @@ def auto_workunit_reporter(report: SourceReport, stream: Iterable[T]) -> Iterabl
 
 def auto_materialize_referenced_tags(
     stream: Iterable[MetadataWorkUnit],
-    active: bool = True,
```

Comment:
Now handled by passing in `None` for the workunit processor, although I don't think this was ever passed as `False`